pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -134,7 +134,7 @@ type KogitoServiceSpec struct {
 	// +listType=atomic
 	// Environment variables to be added to the runtime container. Keys must be a C_IDENTIFIER.
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
-	Envs []corev1.EnvVar `json:"envs,omitempty"`
+	Envs []corev1.EnvVar `json:"env,omitempty"`
```

Comment:
Why not changing the variable as well?

Reply:
Okay, will change the variable name as well :)